src/pyedb/dotnet/edb_core/edb_data/raptor_x_simulation_setup_data.py

```diff
@@ -0,0 +1,448 @@
+
+    @property
+    def net_settings_options(self):
+        """A list of Name, Value pairs that stores advanced option."""
+        return [val for val in list(self._advanced_settings.NetSettingsOptions)]
+
+    @net_settings_options.setter
+    def net_settings_options(self, value):
+        if isinstance(value, list):
+            self._advanced_settings.NetSettingsOptions = convert_py_list_to_net_list(value)
+
+    @property
+    def override_shrink_fac(self):
+        """Set the shrink factor explicitly, that is, review what-if scenarios of migrating to half-node
+        technologies.
+        Units: unitless.
+        """
+        return self._advanced_settings.OverrideShrinkFac
+
+    @override_shrink_fac.setter
+    def override_shrink_fac(self, value):
+        self._advanced_settings.OverrideShrinkFac = EdbValue(value).tofloat
+
+    @property
+    def plane_projection_factor(self):
+        """To eliminate unnecessary mesh complexity of "large" metal planes and improve overall extraction time,
+        user can define the mesh of certain planes using a combination of the Plane Projection Factor and
+        settings of the Nets Advanced Options.
+        Units: unitless.
+        """
+        return self._advanced_settings.PlaneProjectionFactor
+
+    @plane_projection_factor.setter
+    def plane_projection_factor(self, value):
+        self._advanced_settings.PlaneProjectionFactor = EdbValue(value).tofloat
+
+    @property
+    def use_accelerate_via_extraction(self):
+        """Setting this option will simplify/merge neighboring vias before sending the layout for processing
+        to the mesh engine and to the EM engine.
+        """
+        return self._advanced_settings.UseAccelerateViaExtraction
+
+    @use_accelerate_via_extraction.setter
+    def use_accelerate_via_extraction(self, value):
+        if isinstance(value, bool):
+            self._advanced_settings.UseAccelerateViaExtraction = value
+
+    @property
+    def use_auto_removal_sliver_poly(self):
+        """Setting this option simplifies layouts by aligning slightly misaligned overlapping polygons."""
+        return self._advanced_settings.UseAutoRemovalSliverPoly
+
+    @use_auto_removal_sliver_poly.setter
+    def use_auto_removal_sliver_poly(self, value):
+        if isinstance(value, bool):
+            self._advanced_settings.UseAutoRemovalSliverPoly = value
+
+    @property
+    def use_cells_per_wavelength(self):
+        """This setting describes the number of cells that fit under each wavelength. The wavelength is calculated
+        according to the Max Frequency or the Mesh Frequency, unless specified by user through this setting.
+        """
+        return self._advanced_settings.UseCellsPerWavelength
+
+    @use_cells_per_wavelength.setter
+    def use_cells_per_wavelength(self, value):
+        if isinstance(value, bool):
+            self._advanced_settings.UseCellsPerWavelength = value
```

Comment:
Can you log an error if value is not bool ?

Reply:
done